File: Medium/JavaScript/1561. Maximum Number of Coins You Can Get.js
var maxCoins = function(piles) {
    // Sort the array, then initialize a variable to contain the max
    // number of coins you can have, and initialize a variable of piles.length
    piles.sort((a, b) => a - b);
    let max = 0;
    const len = piles.length;

    // Use a for loop to distribute the coins
    for(let i = len / 3; i < len - 1; i += 2){
        max += piles[i];
    }

    // Return the max number of coins you can have
    return max;
};

File: Medium/Python/1561. Maximum Number of Coins You Can Get.py
class Solution(object):
    def maxCoins(self, piles):
        # Sort the list, then initialize a variable to contain the maximum
        # number of coins that you can have, and initialize a variable of len(piles)
        piles.sort()
        maximum = 0
        length = len(piles)

        # Use a for loop to distribute the coins
        for i in range(length / 3, length - 1, 2):
            maximum += piles[i]

        # Return the maximum number of coins you can have
        return maximum
